vm-setup/roles/v1aX_integration_test/tasks/remediation.yml

```diff
@@ -220,6 +220,50 @@
     delay: 20
     register: ready_hosts
     until: ready_hosts.stdout == "1"
+    
+    #### m3dt template reference check
```

Comment:
Remove or change this comment to a more descriptive one

Reply:
Removed. Thanks